osbs/api.py

```diff
@@ -305,7 +302,8 @@ def _create_scratch_build(self, build_request):
             builder_img['kind'] = 'DockerImage'
             builder_img['name'] = ref
 
-        build_config_name = 'scratch-%s' % datetime.datetime.now().strftime('%Y%m%d%H%M%S')
+        build_config_name = 'scratch-%s' % (
+            build_json['spec']['output']['to']['name'][-20:])
```

Comment:
Yes, this was the intent. Updated like recommended to make it easier to understand. Thanks!